### ISSUE TITLE
How to Plot Specific Atlas Regions on Surface? (e.g., Yeo7 Visual Network)

### ISSUE THREAD

**Issue by zhangjb35** [dipterix/threeBrain#39]
Hello,

I would like to plot specific atlas regions on a brain surface. Specifically, I want to visualize the visual network from the Yeo7 atlas. Can I achieve this with threeBrain?

**Comment by dipterix** [comment]
Yes. If you can generate annotation file (e.g. https://surfer.nmr.mgh.harvard.edu/fswiki/CorticalParcellation_Yeo2011), you can directly drag & drop the annotation files to the viewer. 

Please make sure you have the latest version installed either through RAVE (see rave.wiki) or install the dev version.

If you don't have the annotation file, then you can generate one via cortical surface mapping. I will post an example later to this issue.

**Comment by dipterix** [comment]
Hi @zhangjb35 This function has been added to the latest RAVE. Please update RAVE `ravetools` and `threeBrain` package to the latest dev version to use this function:

```r
brain <- raveio::rave_brain("demo/DemoSubject")
brain$add_annotation("label/Yeo2011_7Networks_N1000")
brain$plot(controllers = list("Voxel Data" = "label/Yeo2011_7Networks_N1000"))
```

https://github.com/user-attachments/assets/debc0c8b-44ca-4d26-b101-b1a85ff4516a
